CHANGELOG.md

```diff
@@ -53,6 +53,14 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added logging and custom exit code when app fails to bind the IP address and
   port. (#28)
 
+- Changed version of `github.com/iver-wharf/wharf-core`
+  from v1.1.0 -> v1.2.0. (#30)
+
+- Removed `internal/httputils`, which was moved to `wharf-core/pkg/cacertutil`.
+  (#30)
+
+- Added dependency on `wharf-core/pkg/cacertutil`. (#30)
```

Comment:
nit: Missing full package name. And IMO you can omit these package dependencies. We already have a dedicated release note for the `github.com/iver-wharf/wharf-core` Go module.